src/ca/ubc/cs304/database/tables/Branch.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE Branch(
-    location VARCHAR(255),
-    city VARCHAR(255),
+    location VARCHAR(255) not null,
```

Comment:
Primary keys don't need this, as they are defaulted to this. Also it should be 'NOT NULL' instead of lower case if we do need it.